source/fundamentals/encrypt-fields/qe.txt

```diff
@@ -0,0 +1,80 @@
+.. _node-qe:
+
+==============================
+Configure Queryable Encryption
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to configure the {+driver-short+} for {+qe+}.
+
+For thorough coverage of the core concepts, use-cases, examples, and
+reference, visit the main {+qe+} :ref:`documentation site <qe-manual-feature-qe>`.
+
+Introduction
+------------
+
+{+qe+} allows you to encrypt specific data fields within a document before
+sending the data to the server, and to query on encrypted fields. {+qe+}
+is useful for situations in which applications must guarantee that
+unauthorized parties, including server administrators, cannot read the
+encrypted data.
+
+
+Configuration
+-------------
+
+To use {+qe+}, you must:
+
+- Configure the driver with specific settings for encryption.
+- Configure and use the driver's encryption package, ``mongodb-client-encryption``.
```

Comment:
Issue: potential to miss the connection between these items and the subheaders. Currently references "driver"  and "encryption package" which don't have clear connections to the titles of the subheaders.

Suggestion: Explicitly mention which configuration subheader to which each bullet point is related.